src/OpenTracing/GlobalTracer.php

```diff
@@ -18,7 +18,7 @@
      *
      * @param Tracer $tracer
      */
-    public static function setGlobalTracer(Tracer $tracer)
+    public static function withGlobalTracer(Tracer $tracer)
```

Comment:
I agree with you in general, but for `setGlobalTracer` I don't know any method name that would describe it better